src/Input/Specification.php

```diff
@@ -59,4 +63,22 @@ public function getCompositeResponseFields(): FieldCollection
     {
         return $this->compositeResponseFields;
     }
+
+    public function getAllContentTypes(): array
+    {
+        $allContentTypes = [
+            XmlContentTypeSerializer::MIME_TYPE            => false,
+            FormUrlencodedContentTypeSerializer::MIME_TYPE => false,
+            JsonContentTypeSerializer::MIME_TYPE           => false,
+        ];
+
+        foreach ($this->getOperations() as $operation) {
+            /** @var Operation $operation */
```

Comment:
Pls move the annotation before the first loop. Github actions failure.

Reply:
fixed it, thx